doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -397,16 +397,18 @@ This resource allows you to obtain a list of subresources published by the API:
 
 ## Query Entities [/v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
 
-### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
+### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,georel,geometry,attrs,options}]
 
 + Parameters
     + q: temperature==22.5;humidity==60 (optional, string) - General query parameter that allows
     to filter by attribute content
-    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates. Comma separated values.
-    The number of values will depend on the 'geometry' attribute.
-    + geometry: circle;radius=10000 (optional, string) - Geospatial area for the query. 
-    The 'exterior' modifier allows to indicate that the referred area is the outside
-    of the region, for instance, 'polygon;exterior'
+    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates of a reference geometry used for the query.
+    Comma separated values. The number of values and its interpreation depends on the `geometry` attribute.
```

Comment:
See https://github.com/telefonicaid/fiware-orion/blob/develop/doc/apiary/v2/fiware-ngsiv2-reference.apib#L775

(Not sure when this was introduced, but I guess that using "Blame" buttom and doing some git-archeology based on commit hash you could get the particular branch/PR).

I don't think is redundant. On the contrary, it helps to introduce some structure in the coords field, which makes them easier to read. For example comparing

```
coord=12,11,13,4,5,1,16,17,7,15,10,8,19,20,9,6,18,3,14,2
```

with

```
coord=12,11;13,4;5,1;16,17;7,15;10,8;19,20;9,6;18,3;14,2
```

The former is more difficult to read and debug. In fact, it is difficult to see looking to the first one that you have an even number of numbers.

From a generation difficulty point of view, I think both approaches are equal, i.e. a programmatic client wouldn't have any problem to generate coords in one format or in the other.